src/routes/+page.svelte

```diff
@@ -4,17 +4,18 @@
 	import Button from '$lib/components/ui/button.svelte'
 	import Checkbox from '$lib/components/ui/checkbox.svelte'
 	import Radio from '$lib/components/ui/radio.svelte'
+	import Typography from '$lib/components/ui/typography.svelte'
```

Comment:
This is more like a feature request, but it would be nice to be able to import elements from typography.

For example
`import { H1 } from '$lib/components/ui/typography.svelte`